stdlib/int32.mli

```diff
@@ -52,12 +52,24 @@ external div : int32 -> int32 -> int32 = "%int32_div"
    argument is zero.  This division rounds the real quotient of
    its arguments towards zero, as specified for {!Pervasives.(/)}. *)
 
+val div_unsigned : int32 -> int32 -> int32
+(** Same as [div], except that arguments and result are interpreted as {e
+    unsigned} 32-bit integers.
+
+    @since 4.07.0 *)
```

Comment:
The `@since` indications need to to be updated.

Reply:
Thanks, fixed.